librz/core/cmd_type.c

```diff
@@ -1005,6 +1085,59 @@ RZ_API void rz_core_link_stroff(RzCore *core, RzAnalysisFunction *fcn) {
 	free(buf);
 }
 
+static void print_all_union_format(RzCore *core, Sdb *TDB) {
+	SdbList *l = sdb_foreach_list_filter(TDB, stdifunion, true);
+	SdbListIter *it;
+	SdbKv *kv;
+	ls_foreach (l, it, kv) {
+		type_show_format(core, sdbkv_key(kv), RZ_OUTPUT_MODE_RIZIN);
+	}
+	ls_free(l);
+}
+
+static void type_list_c_all(RzCore *core) {
+	Sdb *TDB = core->analysis->sdb_types;
+	// TODO: Change the logic maybe?
+	//eprintf("Specify the type");
+	//return RZ_CMD_STATUS_ERROR;
+	rz_core_cmd0(core, "tfc");
```

Comment:
I will convert these calls to the API once all those converted to the newshell and new API.